docs/content/concepts/sui-move-concepts/packages/automated-address-management.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Automated Address Management
+---
+
+When you publish or upgrade a package, its address (also known as the package ID) is tracked in the `Move.lock` file. This bookkeeping is done automatically so that you can avoid recording or updating hex addresses (e.g., in the `Move.toml` file). 
```

Comment:
```suggestion
When you publish or upgrade a package, its address (also known as the package ID) is tracked in the `Move.lock` file. This bookkeeping is done automatically so that you can avoid recording or updating hex addresses (for example, in the `Move.toml` file). 
```
Style guide thing. Avoid latin abbreviations for simplicity.